doc/releases/changelog-dev.md

```diff
@@ -22,7 +22,12 @@
 
 * The `diagonalize_measurements` transform now uses a more efficient method of diagonalization
   when possible, based on the `pauli_rep` of the relevant observables.
-  [#6113](https://github.com/PennyLaneAI/pennylane/pull/6113/)
+  [(#6113)](https://github.com/PennyLaneAI/pennylane/pull/6113/)
+
+* An `update` method is added to `QuantumScript` to make it easier to create an updated version
+  of a tape where some of `tape.operations`, `tape.measurements`, `tape.shots` and 
+  `tape.trainable_params` are modified, while ensuring other attributes are unchanged.
```

Comment:
```suggestion
  of a tape where some of `tape.operations`, `tape.measurements`, `tape.shots`, and 
  `tape.trainable_params` are modified while ensuring other attributes are unchanged.
```

It is almost totally irrelevant (just what Grammarly suggests)

Reply:
😅